src/librbd/cache/rwl/InitRequest.h

```diff
@@ -64,6 +71,9 @@ class InitRequest {
   void set_feature_bit();
   void handle_set_feature_bit(int r);
 
+  void shutdown_image_cache();
```

Comment:
Nit: please update the state drawing to include this error path.

Reply:
DONE